log.go

```diff
@@ -3,33 +3,44 @@ package turnpike
 import (
 	"errors"
 	"fmt"
-	glog "log"
+	"log"
 	"os"
 )
 
 var (
-	logFlags = glog.Ldate | glog.Ltime | glog.Lshortfile
-	log      *glog.Logger
+	loggerFlags = log.Ldate | log.Ltime | log.Lshortfile
+	logger      Logger
```

Comment:
It looks like if you don't rename `log` to `logger` you can eliminate a large number of code changes.